packages/next/client/index.js

```diff
@@ -323,14 +329,70 @@ function renderReactElement(reactEl, domEl) {
 
   if (onPerfEntry && ST) {
     try {
-      const observer = new PerformanceObserver(list => {
-        list.getEntries().forEach(onPerfEntry)
-      })
-      // Start observing paint entry types.
-      observer.observe({
-        type: 'paint',
-        buffered: true,
-      })
+      function getSupportedTypes(types) {
```

Comment:
Not sure how much this matters, but I don't think these functions need to be defined within the try/catch block.

Reply:
done